src/gc/unix/gcenv.unix.cpp

```diff
@@ -318,6 +330,25 @@ bool GCToOSInterface::VirtualRelease(void* address, size_t size)
     return (ret == 0);
 }
 
+// Commit virtual memory range.
+// Parameters:
+//  size      - size of the virtual memory range
+//  alignment - requested memory alignment, 0 means no specific alignment requested
+//  flags     - flags to control special settings like write watching
+//  node      - NUMA node
+// Return:
+//  Starting virtual address of the committed range
+void* GCToOSInterface::VirtualCommitLargePages(size_t size, size_t alignment, uint32_t node)
```

Comment:
When I was designing the `GCToOSInterface`, I have intentionally separated the reserve and commit. Instead of introducing this new combined method, I would prefer adding a parameter to VirtualCommit and VirtualReserve specifying whether large pages are requested or not.

Reply:
@Maoni0 has explained the reason for the combined method, so I now understand the reason for having it. But could you please rename it to `VirtualReserveAndCommitLargePages` so that it is clear that it does both?